python/GafferSceneUI/LightEditor.py

```diff
@@ -354,6 +370,51 @@ def __editSelectedCells( self, pathListing ) :
 
 			self.__popup.popup()
 
+	def __toggleBoolean( self, inspectors, inspections ) :
+
+		plugs = [ i.acquireEdit() for i in inspections ]
+		# Make sure all the plugs are either a BoolPlug or contain a BoolPlug
+		if not all (
+			(
+				isinstance( p, ( Gaffer.TweakPlug, Gaffer.NameValuePlug ) ) and
+				isinstance( p["value"], Gaffer.BoolPlug )
+			) or isinstance( p, Gaffer.BoolPlug ) for p in plugs
+		) :
+			return False
+
+		currentValues = []
+
+		# Use a single new value for all plugs.
+		# First we need to find out what the new value would be for each plug in isolation.
+		for inspector, pathInspections in inspectors.items() :
+			for path, inspection in pathInspections.items() :
+				currentValue = inspection.value().value if inspection.value() is not None else None
+
+				if isinstance( inspector, GafferSceneUI.Private.AttributeInspector ) :
+					fullAttributes = self.__settingsNode["in"].fullAttributes( path[:-1] )
+					parentValueData = fullAttributes.get( "light:mute", None )
+					parentValue = parentValueData.value if parentValueData is not None else False
+
+					currentValues.append( currentValue if currentValue is not None else parentValue )
+				else :
+					currentValues.append( currentValue )
+
+		# Now set the value for all plugs, defaulting to `True` if they are not
+		# currently all the same.
+		newValue = not sole( currentValues )
+
+		for inspector, pathInspections in inspectors.items() :
+			for path, inspection in pathInspections.items() :
+				plug = inspection.acquireEdit()
+
+				if isinstance( plug, ( Gaffer.TweakPlug, Gaffer.NameValuePlug ) ) :
+					plug["value"].setValue( newValue )
+					plug["enabled"].setValue( True )
+				else :
+					plug.setValue( newValue )
```

Comment:
This needs to go in an UndoScope.

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/4948/commits/554e1f5e381afa936daf87228cb0e9e800bf2ed2